opendbc/safety/safety_declarations.h

```diff
@@ -13,12 +13,14 @@
   do { \
     (config).rx_checks = (rx); \
     (config).rx_checks_len = sizeof((rx)) / sizeof((rx)[0]); \
+    (config).fwd_buses = NULL; \
```

Comment:
this is needed since we commonly do `safety_config ret;` everywhere which doesn't zero initialize this to NULL.

TODO: change to a forwarding boolean since everything uses 0: 2, 2: 0 except all_output which is based on a dynamic param